### DISCUSSION TITLE
Make notify() api available on tables

### DISCUSSION THREAD

**Discussion by underdpt** [filamentphp/filament#1423] [Ideas]
Hello,

I've been digging a bit, and it seems that `notify()` is only available on `vendor\filament\filament\src\Pages\Page.php` and managed on `vendor\filament\filament\resources\views\components\page.blade.php`. 

Could it be possible to make this available on all Filament pages? For example, this would allow us to use `notify()` inside relation manager actions.

**Comment by danharrin** [comment]
Yeah, I have plans to extract notifications to a separate LW component so that they can be called from anywhere, including RMs.

**Comment by danharrin** [comment]
Done - https://filamentphp.com/docs/2.x/admin/pages#sending-flash-notifications
